README.md

```diff
@@ -75,10 +91,10 @@ When using pyPESTO in your project, please cite
   Bioinformatics, 2023, btad711, [doi:10.1093/bioinformatics/btad711](https://doi.org/10.1093/bioinformatics/btad711)
 
 When presenting work that employs pyPESTO, feel free to use one of the icons in
-[doc/logo/](https://github.com/ICB-DCM/pyPESTO/tree/main/doc/logo):
+[doc/logo/](doc/logo):
 
 <p align="center">
-  <img src="https://raw.githubusercontent.com/ICB-DCM/pyPESTO/master/doc/logo/logo.png" height="75" alt="AMICI Logo">
+  <img src="https://raw.githubusercontent.com/ICB-DCM/pyPESTO/main/doc/logo/logo.png" height="75" alt="AMICI Logo">
```

Comment:
```suggestion
  <img src="https://raw.githubusercontent.com/ICB-DCM/pyPESTO/main/doc/logo/logo.png" height="75" alt="pyPESTO Logo">
```

copy pasta? :)

Reply:
seems like it, I was actually asking myself that 🙈

Reply:
looked it up, definitely copy pasta 🍝